### PR TITLE
Fix/minor issues jan 25

### PR DESCRIPTION
**Pull request by AdityaKhatri**
## Changes

* Show lead preview on click on the title
* Make assignee editable
* Select entry while going from notifications
* Add ability to change no of items in source listing page

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [X] eslint issues
- [x] typescript issues
- [x] codegen errors
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments
- [x] conflict markers

## This PR contains valid:

- [x] permission checks
- [x] translations

### CODE REVIEW

**Review by samshara — APPROVED** [review]

File: app/components/entryReview/EntryComments/Comment/EditCommentForm/index.tsx [hunk 1]
```diff
@@ -29,6 +33,7 @@ type FormSchemaFields = ReturnType<FormSchema['fields']>;
 const schema: FormSchema = {
     fields: (): FormSchemaFields => ({
         text: [requiredStringCondition],
+        mentionedUsers: [requiredCondition],
```
Comment (tnagorra) [inline app/components/entryReview/EntryComments/Comment/EditCommentForm/index.tsx]: We may need to use requiredListCondition

File: app/components/entryReview/EntryComments/Comment/EditCommentForm/index.tsx [hunk 2]
```diff
@@ -45,12 +51,20 @@ function EditCommentForm(props: Props) {
         comment,
         onEditSuccess,
         onEditCancel,
+        projectId,
     } = props;
 
     const [initialFormValue] = useState<FormType>({
         text: comment.textHistory[0]?.text ?? '',
+        mentionedUsers: comment.mentionedUsers.map((u) => String(u)),
```
Comment (tnagorra) [inline app/components/entryReview/EntryComments/Comment/EditCommentForm/index.tsx]:
```suggestion
        mentionedUsers: comment.mentionedUsers.map(String),
```

File: app/components/lead/LeadPreviewButton/index.tsx [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+            lead (id: $leadId) {
+                id
+                title
+                url
+                attachment {
+                    id
+                    title
+                    mimeType
+                    file {
+                        url
```
Comment (tnagorra) [inline app/components/lead/LeadPreviewButton/index.tsx:34]: Let's fetch `id` for file

File: app/components/lead/LeadPreviewButton/index.tsx [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+                            url={leadData?.url ?? undefined}
+                            attachment={leadData?.attachment ?? undefined}
+                        />
+                    ) : (
+                        <Message
+                            icon={<Kraken variant="sleep" />}
+                            pending={leadLoading}
+                            message="No preview available"
+                        />
+                    )}
```
Comment (tnagorra) [inline app/components/lead/LeadPreviewButton/index.tsx:128]: Why not handle loading state here?

**Review by tnagorra — DISMISSED** [review]

File: app/components/lead/LeadPreviewButton/index.tsx (hunk 4 above)
Comment (AdityaKhatri) [reply app/components/lead/LeadPreviewButton/index.tsx:128]: it is handled in Message component

**Review by samshara — APPROVED** [review]

**Pull request merged**